docs/qml/qaoa.md

```diff
@@ -34,57 +36,111 @@ The goal of the MaxCut algorithm is to maximize the following cost function:
 
 $$\mathcal{C}(p) = \sum_{\alpha}^m \mathcal{C}_{\alpha}(p)$$
 
-where $p$ is the given partition of the graph, $\alpha$ is an index over the edges and $\mathcal{C}_{\alpha}(p)$ is written
+where $p$ is a given cut of the graph, $\alpha$ is an index over the edges and $\mathcal{C}_{\alpha}(p)$ is written
 such that if the nodes connected by the $\alpha$ edge are in the same set, it returns $0$, otherwise it returns $1$.
 
+Since in this tutorial we are only dealing with small graphs, we can find the maximum cut by brute force to make sure QAOA works as intended. Recall that every cut is represented by two bitstrings, e.g. cuts "0011" and "1100" are equivalent as they represent the same cut.
```

Comment:
What is the meaning of the bitstrings here exactly? Is the number of bits equal to the number of edges?

Reply:
The number of bits is equal to the number of nodes. With bitstrings I refer to binary strings representing each possible solution (each possible "cut" of the graph), assigning "0" to bit $i$ if node $i$ is in partition A and "1" if it is in partition "B" (or viceversa). I will elaborate more on this so it's more clear!

Reply:
Got it, thanks.